bower.json

```diff
@@ -23,6 +23,6 @@
         "tests"
     ],
     "dependencies": {
-        "polymer": "Polymer/polymer"
+        "polymer": "Polymer/polymer#1.9 - 2"
```

Comment:
Maybe we could add a note to the README, that it is a hybrid element, so it will be clear to everybody